plugin/trino-delta-lake/README.md

```diff
@@ -0,0 +1,179 @@
+# Delta Lake Connector Developer Notes
```

Comment:
Squash `Document setup of Databricks Delta Lake product test environment` 
with `Test Delta Lake connector against Databricks runtime`